lib/workers/repository/init/vulnerability.ts

```diff
@@ -78,75 +74,43 @@ export async function detectVulnerabilityAlerts(
   };
   const combinedAlerts: CombinedAlert = {};
   for (const alert of alerts) {
-    if (
-      alert.securityVulnerability?.package?.name === 'yargs-parser' &&
-      (alert.vulnerableRequirements === '= 5.0.0-security.0' ||
-        alert.vulnerableRequirements === '= 5.0.1')
-    ) {
-      continue;
-    }
```

Comment:
why removed?

Reply:
We do not get the `vulnerableRequirements` field in the api response anymore.